src/WorldStorage/WorldStorage.cpp

```diff
@@ -167,8 +167,8 @@ void cWorldStorage::QueueSaveChunk(int a_ChunkX, int a_ChunkZ)
 void cWorldStorage::InitSchemas(int a_StorageCompressionFactor)
 {
 	// The first schema added is considered the default
-	m_Schemas.push_back(new cWSSAnvil    (m_World, a_StorageCompressionFactor));
-	m_Schemas.push_back(new cWSSForgetful(m_World));
+	m_Schemas.emplace_back(new cWSSAnvil    (m_World, a_StorageCompressionFactor));
```

Comment:
I think all the pointers should remain push_back, ref http://stackoverflow.com/questions/15783342/should-i-use-c11-emplace-back-with-pointers-containers

Reply:
I thought the lists all contained smart pointers. But yeah - if we push back normal pointers you're right

Reply:
There are actually some smart pointer lists, arrays,...

e.g. the BioGen caches